tools/reaper/internal/internal.go

```diff
@@ -127,16 +128,17 @@ func ReapNamespaces(params *Params) error {
 	}
 
 	for _, namespace := range namespaces.Items {
-		if isOrphaned(namespace, params) {
-			if err = namespaceInterface.Delete(namespace.ObjectMeta.Name, &metav1.DeleteOptions{}); err != nil {
-				// Log but dont panic the error
-				log.Printf("Error: deleting namespace %s created at %s", namespace.ObjectMeta.Name, namespace.ObjectMeta.CreationTimestamp)
-			} else {
-				log.Printf("deleting namespace %s created at %s", namespace.ObjectMeta.Name, namespace.ObjectMeta.CreationTimestamp)
-			}
-		} else {
+		if !isOrphaned(namespace, params) {
 			log.Printf("skipping namespace %s created at %s", namespace.ObjectMeta.Name, namespace.ObjectMeta.CreationTimestamp)
+			continue
 		}
+		if err = namespaceInterface.Delete(namespace.ObjectMeta.Name, &metav1.DeleteOptions{}); err != nil {
```

Comment:
err isn't created here, so I don't think it makes sense to use this form of a if statement (as nothing about scoping changes)

Just set err on one line, and check it on the next.